pyefis.xml

```diff
@@ -1,147 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Phil a form of this file got move to the GWFix program since the polling is going to be coming from there?